test/issues/279-wrong-canvas-size.js

```diff
@@ -0,0 +1,54 @@
+(function() {
+	var WIDTH = 600;
+	var HEIGHT = 400;
+	function buildKernel(mode) {
+		var gpu = new GPU({ mode });
+
+		
+		var body;
+
+		var initMatrix = gpu.createKernel(function(value) {  
+			return value;
+		})
+			.setOutput([WIDTH, HEIGHT]);
+
+		var render = gpu.createKernel(function(matrix) {
+			var i = matrix[this.thread.y][this.thread.x];
+			this.color(i, i, i, 1);
+		})
+			.setOutput([WIDTH, HEIGHT])
+			.setGraphical(true);
+
+		var matrix = initMatrix(0.5);
+		render(matrix);
+		return render;
+	}
+	
+	QUnit.test('Issue #314 Large array addressing - auto', () => {
+		
+		var canvas = buildKernel().getCanvas();
+		QUnit.assert.equal(canvas.width, WIDTH);
+		QUnit.assert.equal(canvas.height, HEIGHT);
+	});
+
+	// QUnit.test('Issue #314 Large array addressing - gpu', () => {
```

Comment:
Was this meant to be left in?

Reply:
yes, and the title should not be 'Issue #314 Large array addressing'

will fix